config/environment.js

```diff
@@ -51,5 +64,17 @@ module.exports = function (environment) {
     // here you can enable a production-specific feature
   }
 
+  if (environment === 'production') {
+    // TODO set the proper value for production
```

Comment:
This TODO comment isn't needed right, since it will be configured on the server?